data/json/itemgroups/guns.json

```diff
@@ -227,6 +227,8 @@
       { "item": "h&k416a5", "prob": 50, "charges-min": 0, "charges-max": 30 },
       { "item": "m107a1", "prob": 30, "charges-min": 0, "charges-max": 10 },
       { "item": "as50", "prob": 20, "charges-min": 0, "charges-max": 10 },
+      { "item": "tac50", "prob": 20, "charges-min": 0, "charges-max": 5 },
+      { "item": "ar50", "prob": 20, "charges-min": 0, "charges-max": 1 },
```

Comment:
The AR-50 probably belongs in a civilian rifle group, and maybe a very rare spawn in the swat group. The Tac 50 should be pretty rare, about 5 or 7 maybe, and also show up rarely in the swat group.

Also, why is the AS50 showing up here? That should pretty much only be in the Swat/paramilitary group. The m107 covers the military's semi-auto AMR needs.

Reply:
so i should put AR-50 on "guns_rifle_common" category with probability of 30-40 and "guns_rifle_rare" category with probability of 20-30? and TAC-50 too with probability of less than 20?

according to internet, AS50 is being used by Cyprut national guard so i put it on Milspec category (and already merged at build 9660)

Reply:
No US branches appear to use the AS50, so it does not belong. This sort of thing is fine for fatcat security and police agencies, albeit again, very unusual and rare.

The seals appear to occasionally use [the AW-50](https://books.google.com/books?id=M9fvCwAAQBAJ&pg=PT74&lpg=PT74&dq=aw50+seals+.50&source=bl&ots=TaMRrVZGCc&sig=ACfU3U0qqOu6nzD9ZvyX4E8eVycRpYELVg&hl=en&sa=X&ved=2ahUKEwik6sLYm-nkAhUFGKwKHXoNACkQ6AEwEnoECAgQAQ#v=onepage&q=aw50%20seals%20.50&f=false), but not the AS-50.

Based on results on gunbroker : an AR-50 is about 7.63% as common as an M1a; therefore in guns rifles common, it should be about 4.

I don't think even 1 would be appropriate for the TAC-50; Other .50bmg rifles are better commercial successes in the civ market.

Reply:
4 by mean of probability of 40? AS50 was requested by forum users so i made it and it got merged.

i will try to put it first and see what needs to be changed

Reply:
No, literally 4. As in
`      { "item": "ar50", "prob": 4, "charges-min": 0, "charges-max": 1 },`

Reply:
are you sure? i see that on the itemgroup every weapon probability is factor of 5 (5, 10, 15,20.. 100) and in percentage, so 4 means only 4% spawn?

i don't know about that, but all i know is that if its merged it means approved by many.

Reply:
I am certain.

Reply:
3.6 rounded become 4 right? if that inserted into itemgroup prob (which is factor of 5) that means 20. which is the prob i already inserted into the json.